src/site/content/en/blog/local-fonts/index.md

```diff
@@ -0,0 +1,348 @@
+    const sfnt = await metadata.blob();
+
+    const sfntVersion = (new TextDecoder).decode(
+        // Slice out only the bytes we need: the first 4 bytes are the SFNT
+        // version info.
+        // Spec: https://docs.microsoft.com/en-us/typography/opentype/spec/otff#organization-of-an-opentype-font
+        await sfnt.slice(0, 4).arrayBuffer());
+
+    let outlineFormat = 'UNKNOWN';
+    switch (sfntVersion) {
+      case '\x00\x01\x00\x00':
+      case 'true':
+      case 'typ1':
+        outlineFormat = 'truetype';
+        break;
+      case 'OTTO':
+        outlineFormat = 'cff';
+        break;
+    }
+    console.log('Outline format:', outlineFormat);
+  }
+} catch (err) {
+  console.error(err.name, err.message);
+}
+```
+
+## Demo
+
+You can see the Local Font Access API in action in the
+[demo](https://local-font-access.glitch.me/demo/) below.
+Be sure to also check out the
+[source code](https://glitch.com/edit/#!/local-font-access?path=README.md%3A1%3A0).
+The demo showcases a custom element `<font-select>` that implements a local font picker.
+
+<div class="glitch-embed-wrap" style="height: 500px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/local-font-access?path=index.html&previewSize=100"
+    title="local-font-access on Glitch"
+    allow="local-fonts"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+## Privacy considerations
+
+The `"local-fonts"` permission appears to provide a highly fingerprintable surface.
+However, browsers are free to return anything they like.
+For example, anonymity-focused browsers may choose to only provide
+a set of default fonts built into the browser.
+Similarly, browsers are not required to provide table data exactly as it appears on disk.
+
+Wherever possible, the Local Font Access API is designed to only expose exactly the information
+needed to enable the mentioned use cases.
+System APIs may produce a list of installed fonts not in a random or a sorted order,
+but in the order of font installation.
+Returning exactly the list of installed fonts given by such a system API
+can expose additional entropy bits, and use cases we want to enable
+are not assisted by retaining this ordering.
+As a result, this API requires that the returned data be sorted before being returned.
+
+## Security and permissions
+
+The Chrome team has designed and implemented the Local Font Access API using the core principles defined in [Controlling Access to Powerful Web Platform Features][powerful-apis], including user control, transparency, and ergonomics.
+
+### User control
+
+Access to a user's fonts is fully under their control and will not be allowed unless the `"local-fonts"` permission, as listed in the
+[permission registry](https://w3c.github.io/permissions/#permission-registry), is granted.
+
+### Transparency
+
+The fact whether a site has been granted access to the user's local fonts will be visible in the *site information* sheet.
```

Comment:
What's a site information sheet? Maybe add a link.

Reply:
Is that the thing that pops up when you click the lock icon in the address bar? Maybe show a picture instead.

Reply:
It is, yes. For clarity, I have added a link: https://support.google.com/chrome/answer/114662?hl=en&co=GENIE.Platform=Desktop.